README.md

```diff
@@ -191,10 +191,21 @@ class Media extends Resource
     }
 }
 ```
-
 > When you have defined a Larupload attachment entity in your model, you should avoid using the same name for your Nova fields as the entity. As you can see in the code above, the second argument of the make function is `videos`, not `video`.
 
-> Larupload has its own disk, so the third argument of make function (disk) is not used when you are using larupload to handle upload process.  
+> Larupload has its own disk, so the third argument of make function (disk) is not used when you are using larupload to handle upload process.
+
+
+
+5- Configure`Larupload`to use the same disk as `NovaVideo`.
```

Comment:
When you use `storeWithLarupload`, the entire uploading process will be handled by Larupload.
This means that the disk can be set using the [Attachment](https://mostafaznv.gitbook.io/larupload/advanced-usage/attachment/disk) item or the Larupload [config](https://mostafaznv.gitbook.io/larupload/advanced-usage/configuration/disk) file. It doesn't need to be the same disk as `NovaVideo`. Therefore, I believe this part is incorrect.